src/app/content/hooks/receiveContent.ts

```diff
@@ -42,10 +35,9 @@ const hookBody: ActionHookBody<typeof receivePage> = ({
   }
 
   const title = createTitle(page, book);
-
-  // the abstract could be '<div/>'.
-  const abstract = stripHtmlAndTrim(page.abstract ? page.abstract : '');
-  const description = abstract || stripHtmlAndTrim(getCleanContent(book, page, archiveLoader));
+  const cleanContent = getCleanContent(book, page, archiveLoader);
```

Comment:
i think it'd be slightly cleaner to put the `getCleanContent` inside `createDescription` unless there is some other use for it out here